### DISCUSSION TITLE
Some page content not accessible to the model

### DISCUSSION THREAD

**Discussion by tristanhervouet** [n4ze3m/page-assist#266] [General]
Hi,

Thank you so much for your extension, that seems very promising.
I'm facing a problem when asking the AI model about a web page content (tested with Llama 3.2 & Mistral latest). It seems that on some websites it cannot access to a part of the page content. 

For example, on this page: https://www.joomunited.com/ the 2 lines of blocks in the middle of the page called "Featured..." in purple color. When I ask the model about it, it cannot find it, even a block related CSS class. That's weird because for the rest it works, and when asking about it using the Ollama webUI with the page URL it works. 
It also happens on other pages with regular HTML content.

Thanks!

**Comment by n4ze3m** [comment]
Hey, thanks for reporting this! Our current website parsing isn't great at all. I’ll work on improving it. :)

**Comment by tristanhervouet** [comment]
Thanks for the feedback! I just wanted to make sure there was not a setting to change :)

**Reply by n4ze3m** [reply]
There is an option. By default, Page Assist uses the RAG method. You can turn it off, and it will then pass 4096 chunks to the LLM. For some use cases, this can be really effective

![image](https://github.com/user-attachments/assets/aa9b9899-3465-4af5-9931-0e6002089ee8)
